.github/workflows/self-push.yml

```diff
@@ -78,6 +78,61 @@ jobs:
           name: run_all_tests_torch_gpu_test_reports
           path: reports
 
+  run_tests_flax_gpu:
+    runs-on: [self-hosted, docker-gpu, single-gpu]
+    container:
+      image: tensorflow/tensorflow:2.4.1-gpu
+      options: --gpus 0 --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
+    steps:
+      - name: Install dependencies
+        run: |
+          apt -y update && apt install -y software-properties-common && apt -y update && add-apt-repository -y ppa:git-core/ppa && apt -y update && apt install -y git
+          pip install --upgrade "jax[cuda111]" -f https://storage.googleapis.com/jax-releases/jax_releases.html
+          pip install --upgrade pip
+          pip install .[sklearn,testing,sentencepiece,flax,flax-speech,vision]
+
+      - name: Launcher docker
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: NVIDIA-SMI
+        run: |
+          nvidia-smi
+
+      - name: Are GPUs recognized by our DL frameworks
+        run: |
+          python -c "from jax.lib import xla_bridge; print('GPU available:', xla_bridge.get_backend().platform)"
+          python -c "import jax; print('Number of GPUs available:', len(jax.local_devices()))"
+      
+#      - name: Fetch the tests to run
+#        run: |
+#          python utils/tests_fetcher.py --diff_with_last_commit | tee test_preparation.txt
+
```

Comment:
This should be uncommented or all the tests will be run.

Reply:
Sorry - I already resolved this comment: https://github.com/huggingface/transformers/pull/13313#discussion_r698253376 - I'm fine either way. Think it'll definitely slow down the circle ci commits to master a bit if we leave it commented out...What do you think @LysandreJik @sgugger ?

Reply:
I think it's fine to have all tests running on every commit the first couple of days to ensure that everything runs smoothly, especially regarding flaky tests since we're adding a bunch at a time